CHANGELOG.md

```diff
@@ -15,7 +16,7 @@
   * 3.1.0 [\#5170](https://github.com/rvm/rvm/pull/5170),
   * 3.1.1 [\#5190](https://github.com/rvm/rvm/pull/5190)
   * 2.7.6, 3.0.4 and 3.1.2 [\#5207](https://github.com/rvm/rvm/pull/5207)
-  * 2.7.7, 3.0.5, 3.1.3, 3.2.0 Preview 2 and 3.2.0 Preview 3 [\#5271](https://github.com/rvm/rvm/pull/5271)
```

Comment:
Hmm, the patches for `3.2.0 Preview 2` are gone?

Reply:
You should rather keep references to both PR here..

Reply:
The original CHANGELOG line from the previous commit was incorrect. It added entries fro unpatched ruby in the railsexpress section. I did not spot that.